src/models/book.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from database import Base
+from sqlalchemy import DateTime, Integer, String, func
+from sqlalchemy.orm import Mapped, mapped_column
+
+
+class Books(Base):
+    """
+    A class representing a book in the database.
+
+    Attributes:
+        id (int): The primary key ID of the book.
+        title (str): The title of the book.
+        date_of_publication (datetime): The date of publication of the book.
+        isbn (str): The ISBN number of the book.
+        description (str): A description of the book.
+        language_id (int): The ID of the language the book is written in.
+        created_at (datetime): The date and time the book was created.
+        updated_at (datetime): The date and time the book was last updated.
+    """
+
+    __tablename__ = "books"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    title: Mapped[str] = mapped_column(String(32), unique=True, nullable=False)
```

Comment:
Isn't 32 a very short length for title? I think titles can be lengthier than that...

Also we don't necessarily need to provide lengths imo

Reply:
Talha suggested 32 for strings ! I ll fix this .